index.html

```diff
@@ -215,11 +215,11 @@ <h2>
       </p>
       <p>
         The term <dfn>navigation</dfn> refers to the act of <a data-cite=
-        "!HTML/browsing-the-web.html#navigate">navigating</a>.
+        "HTML/browsing-the-web.html#navigate">navigating</a>.
       </p>
       <p>
         The term <dfn>current document</dfn> refers to the document associated
-        with the <a data-cite="!HTML/dom.html#document">Window
+        with the <a data-cite="HTML/dom.html#document">Window
```

Comment:
I think this one is problematic... you are citing some kind of idea about a "the Window objects newest Document object". So, either link to that concept explicitly, or change this to link to both Window and Document in the HTML spec.

Reply:
I agree that this is problematic, but let's land this as a followup. I think we can rewrite it to be the "current browsing context's active document", but that seems to go beyond the scope of this mostly-editorial PR